dashboard.csss-ci.com/server/api/controllers/rest/v1/Proposals.php

```diff
@@ -246,6 +288,11 @@ public function submission_get($id) {
         }
     }
 
+    /**
+     * Finds the desired need id in the proposals model
```

Comment:
please provide more context. For example this would be "find out what a particular proposal's needs are"